tests/unit/amo/components/TestMoreAddonsByAuthorsCard.js

```diff
@@ -0,0 +1,456 @@
+import * as React from 'react';
+
+import MoreAddonsByAuthorsCard, {
+  MoreAddonsByAuthorsCardBase,
+} from 'amo/components/MoreAddonsByAuthorsCard';
+import {
+  fetchAddonsByAuthors,
+  loadAddonsByAuthors,
+} from 'amo/reducers/addonsByAuthors';
+import {
+  ADDON_TYPE_EXTENSION,
+  ADDON_TYPE_DICT,
+  ADDON_TYPE_LANG,
+  ADDON_TYPE_OPENSEARCH,
+  ADDON_TYPE_THEME,
+} from 'core/constants';
+import AddonsCard from 'amo/components/AddonsCard';
+import {
+  dispatchClientMetadata,
+  fakeAddon,
+  fakeAuthor,
+} from 'tests/unit/amo/helpers';
+import {
+  createStubErrorHandler,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+
+
+describe(__filename, () => {
+  function fakeAddons() {
+    const firstAddon = {
+      ...fakeAddon,
+      id: 6,
+      authors: [
+        { ...fakeAuthor, username: 'test', id: 51 },
+        { ...fakeAuthor, username: 'test2', id: 61 },
+      ],
+    };
+    const secondAddon = {
+      ...fakeAddon,
+      id: 7,
+      authors: [{ ...fakeAuthor, username: 'test2', id: 61 }],
+    };
+    const thirdAddon = {
+      ...fakeAddon,
+      id: 8,
+      authors: [{ ...fakeAuthor, username: 'test3', id: 71 }],
+    };
+
+    return { firstAddon, secondAddon, thirdAddon };
+  }
+
+  function addonsWithAuthorsOfType({ addonType, multipleAuthors = false }) {
+    return loadAddonsByAuthors({
+      addons: [
+        {
+          ...fakeAddon,
+          id: 5,
+          slug: 'foo',
+          type: addonType,
+          authors: [fakeAuthor],
+        },
+        {
+          ...fakeAddon,
+          id: 6,
+          slug: 'bar',
+          type: addonType,
+          authors: multipleAuthors ?
+            [{ ...fakeAuthor, id: 50000, username: 'secondAuthor' }] :
+            [fakeAuthor],
+        },
+      ],
+      addonType,
+      authorNames: multipleAuthors ?
+        [fakeAuthor.username, 'secondAuthor'] : [fakeAuthor.username],
+    });
+  }
+
+  function render(customProps = {}) {
+    const props = {
+      i18n: fakeI18n(),
+      store: dispatchClientMetadata().store,
+      ...customProps,
+    };
+
+    return shallowUntilTarget(
+      <MoreAddonsByAuthorsCard {...props} />,
+      MoreAddonsByAuthorsCardBase
+    );
+  }
+
+  it('should render a card', () => {
+    const { store } = dispatchClientMetadata();
+    const authorNames = ['test', 'test2', 'test3'];
```

Comment:
I feel like it's dangerous to hard code these values here, and also have them hardcoded in `fakeAddons()` above, as they could possibly get out of sync. I think it would be better to have this as a high level `const` and then use the values of that const in `fakeAddons()` which would assure that the values are always the same in both places.

Reply:
Sounds good 👍